apps/builder/app/builder/features/settings-panel/props-section/use-props-logic.ts

```diff
@@ -152,6 +153,28 @@ export const usePropsLogic = ({
   updateProp,
   deleteProp,
 }: UsePropsLogicInput) => {
+  const isContentMode = useStore($isContentMode);
+
+  /**
+   * In content edit mode we show only Image and Link props
+   * In the future I hope the only thing we will show will be Components
+   */
+  const isPropVisible = (propName: string) => {
+    const contentModeWhiteList: Partial<Record<string, string[]>> = {
+      Image: ["src", "width", "height", "alt"],
+      Link: ["href"],
+      RichTextLink: ["href"],
```

Comment:
don't like targeting component names, we need to make it configurable from sdk in the future, maybe add todo?

Reply:
def. need to change this before launching/considering it done